benchmarking/instance_test.go

```diff
@@ -0,0 +1,51 @@
+package benchmarking
+
+import (
+	"bytes"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	wazero_runtime "github.com/ChainSafe/gossamer/lib/runtime/wazero"
+	"github.com/ChainSafe/gossamer/lib/trie"
+	sc "github.com/LimeChain/goscale"
+	primitives "github.com/LimeChain/gosemble/primitives/types"
+	ctypes "github.com/centrifuge/go-substrate-rpc-client/v4/types"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	keyTimestampHash, _    = common.Twox128Hash([]byte("Timestamp"))
+	keyTimestampNowHash, _ = common.Twox128Hash([]byte("Now"))
+)
+
+func TestInstance(t *testing.T) {
+	b := &testing.B{}
+	runtime := wazero_runtime.NewBenchInstanceWithTrie(b, "../build/runtime.wasm", trie.NewEmptyTrie())
+	defer runtime.Stop()
+
+	instance, err := newBenchmarkingInstance(runtime, 1)
+	assert.NoError(t, err)
+
+	(*instance.Storage()).Put(append(keyTimestampHash, keyTimestampNowHash...), sc.U64(0).Bytes())
+
+	nowStorageValue, err := sc.DecodeU64(bytes.NewBuffer((*instance.Storage()).Get(append(keyTimestampHash, keyTimestampNowHash...))))
+	assert.NoError(b, err)
+	assert.Equal(b, sc.U64(0), nowStorageValue)
+
+	now := uint64(time.Now().UnixMilli())
+
+	err = instance.ExecuteExtrinsic(
+		"Timestamp.set",
```

Comment:
move to `runtime/benchmark_timestamp_set_test.go`

Reply:
this is testing the instance struct